### PR TITLE
[MRG+1] Reword whats new

### PR DESCRIPTION
**Pull request by massich**

### CODE REVIEW

**Comment by massich** [comment]
+1

File: doc/whats_new.rst [hunk 1]
```diff
@@ -108,7 +109,7 @@ Changelog
 Bug
 ~~~
 
-- Fix: bug with scaling of data in func:`mne.cov._compute_covariance_auto` by `David Sabbagh `_ 
+- Fix bug with scaling of data in ``mne.cov._compute_covariance_auto`` that was affecting the :class:`mne.decoding.SPoC` estimator by `David Sabbagh`_
```
Comment (dengemann) [inline doc/whats_new.rst]: @massich the :func: is missing here no?

**Comment by agramfort** [comment]
No. This is a private function not listed in api page. The link will not work.

**Comment by dengemann** [comment]
> No. This is a private function not listed in api page. The link will not work.

Ok then!

**Comment by larsoner** [comment]
Thanks @massich

**Pull request merged**